metadata_service/proxy/atlas_proxy.py

```diff
@@ -110,10 +111,10 @@ def _extract_info_from_uri(self, *, table_uri: str) -> Dict:
         result = pattern.match(table_uri)
         return result.groupdict() if result else dict()
 
-    def _parse_reader_qn(self, reader_qn: str) -> Dict:
+    def _parse_bookmark_qn(self, bookmark_qn: str) -> Dict:
```

Comment:
I will not remove this `_parse_reader_qn` function, and create another for bookmarks, since it will be used for the user profile and frequent users.

Reply:
good catch, revereted deleting of this one